etc/cmake/options.cmake

```diff
@@ -250,6 +250,7 @@ ot_option(OT_PLATFORM_LOG_CRASH_DUMP OPENTHREAD_CONFIG_PLATFORM_LOG_CRASH_DUMP_E
 ot_option(OT_PLATFORM_NETIF OPENTHREAD_CONFIG_PLATFORM_NETIF_ENABLE "platform netif")
 ot_option(OT_PLATFORM_POWER_CALIBRATION OPENTHREAD_CONFIG_PLATFORM_POWER_CALIBRATION_ENABLE "power calibration")
 ot_option(OT_PLATFORM_UDP OPENTHREAD_CONFIG_PLATFORM_UDP_ENABLE "platform UDP")
+ot_option(OT_RAM OPENTHREAD_CONFIG_MAC_RADIO_AVAILABILITY_MAP_ENABLE "radio availability map")
```

Comment:
Do we need to have a separate config for this? Can we just have it as part of P2P?